scss/_patterns_form-validation.scss

```diff
@@ -18,261 +18,110 @@
     margin-top: -$sp-unit;
   }
 
-  // Theming
-  @if ($theme-default-forms == 'dark') {
+  .is-caution {
     .p-form-validation__input {
-      .is-success & {
-        @include vf-form-validation-input-success-dark-theme;
-      }
+      background-color: $colors--theme--background-caution-default;
+      border-bottom-color: $colors--theme--border-caution;
 
-      .is-caution & {
-        @include vf-form-validation-input-caution-dark-theme;
+      &:hover {
+        background-color: $colors--theme--background-caution-hover;
       }
 
-      .is-error & {
-        @include vf-form-validation-input-error-dark-theme;
+      &:focus {
+        background-color: $colors--theme--background-caution-active;
```

Comment:
Build is failing on missing variable. There was some typo on colors variables Seulkee already fixed, so rebasing on main branch and doublechecking if var names are correct should hopefully fix the build errors.